README.md

```diff
@@ -24,25 +24,37 @@ The following use-cases motivate this work:
 4. Measuring layout for content not visible on screen
 5. Optimizing single-page-app transition performance
 
+## Quick links
+
+* [`subtree-visibility` spec draft](https://wicg.github.io/display-locking/index.html)
+* [`contain-intrinsic-size` explainer](https://github.com/WICG/display-locking/blob/master/explainer-contain-intrinsic-size.md)
+* [`beforematch` explainer](https://github.com/WICG/display-locking/blob/master/explainer-beforematch.md)
+
 ## Quick summary
 
-In the below, "invisible to rendering/hit testing" means not drawn or returned from any hit-testing algorithms. "not drawn" is in the same sense as `visibility: hidden`, where the content still conceptually has layout sizes, but is invisible to the user.
+In the below, "invisible to rendering/hit testing" means not drawn or returned
+from any hit-testing algorithms. "not drawn" is in the same sense as
+`visibility: hidden`, where the content still conceptually has layout sizes, but
```

Comment:
Turns out visibility: hidden also hides from elementFromPoint and other hit testing algorithms. So we can say that "invisible to rendering/hit testing" means the same as it does for "visibility: hidden" here and in the spec.

Reply:
Same goes for the "layout sizes" comment.